code/game/objects/items/robot/items/inflatable_tool.dm

```diff
@@ -0,0 +1,44 @@
+/obj/item/inflatable/cyborg
+	name = "inflatable wall"
+	desc = "A folded membrane which rapidly expands into a large cubical shape on activation."
+	var/power_use = 400
+	var/structure_type = /obj/structure/inflatable
+	/// How long it takes to inflate
+	var/delay = 1 SECONDS
```

Comment:
1. I see that over the downstream it has no delay to it, did you change your mind after seeing it in action?

2. This value never varies and could become a define instead, unless someone thinks the door should have a different time than the wall.

Reply:
Yes, I immediately uploaded the antispam fix here.

Reply:
Understood